query-engine/core/src/query_ast/write.rs

```diff
@@ -16,14 +16,14 @@ pub enum WriteQuery {
 }
 
 impl WriteQuery {
-    pub fn inject_id(&mut self, record_id: RecordIdentifier) {
+    pub fn inject_id_into_args(&mut self, record_id: RecordIdentifier) {
```

Comment:
Changed the name to make obvious into which part of the struct stuff gets injected.